esignet-default.properties

```diff
@@ -480,7 +480,7 @@ mosip.esignet.vci.key-values={\
           },\
    'latest' : {\
               'credential_issuer': '${mosip.esignet.vci.identifier}', 	\
-              'credential_endpoint': '${mosipbox.public.url}${server.servlet.path}/vci/credential', \
+              'credential_endpoint': '${mosip.esignet.host}${server.servlet.path}/vci/credential', \
```

Comment:
Pls change this to ${mosip.esignet.domain.url}. kinldy check the below line https://github.com/mosip/mosip-config/blob/collab1/esignet-default.properties#L269

Reply:
URL Modified.